x-pack/plugins/integrations_manager/server/routes.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import {
+  ID,
+  API_INTEGRATIONS_LIST,
+  API_INTEGRATIONS_INFO,
+  API_INTEGRATIONS_FILE,
+  API_SAVED_OBJECTS_DETAIL,
+  API_SAVED_OBJECTS_ROOT,
+  SAVED_OBJECT_TYPE,
+} from '../common/constants';
+import { Request } from '../common/types';
+import { fetchInfo, fetchList, getZipInfo } from './registry';
+import { getClient } from './saved_objects';
+
+interface PostRequest extends Request {
+  payload: {
+    body: string;
+  };
+}
+
+// Manager public API paths (currently essentially a proxy to registry service)
+export const routes = [
```

Comment:
Is it planned to add validation to the request params?

Reply:
Yes definitely. I wonder if our work with io-ts will happen soon enough that we can share those lessons here, but I don't want to get stuck on that at this point in the integrations manager work. We can always start out with `Joi` if we need to and convert over later.

Reply:
io-ts is already used in kibana, so it's available right now

Reply:
Oh good, I didn't realize it was already in Kibana. I still think that I need to get up to speed with it more and get comfortable with the paradigm before I can ask other new folks to implement it. 👍

Reply:
Created https://github.com/elastic/kibana/issues/38582 to track this